app/services/users/invitation_service.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+module Users
+  class InvitationService < ApplicationService
+    # @attr_reader params [Hash]
+    # - first_name: [String] new user first name
+    # - last_name: [String] new user last name
+    # - phone: [string] new user phone number
+    # - member_type: [integer] user relationship
+    # - current_user: [User] Current user
+
+    def call
+      invite = User.invite!(invite_params) { |u| u.skip_invitation = true }
+      if invite.errors.empty?
+        update_fields(invite)
+        Twilio::TwilioTextMessenger.perform(twilio_params(invite))
+        return OpenStruct.new(success?: true,
+                              data: { invite_token: invite.raw_invitation_token,
+                                      user: invite.present.page_context },
+                              errors: nil)
+      else
+        return OpenStruct.new(success?: false, data: nil, errors: invite.errors)
+      end
+    end
+
+    private
+
+    def update_fields(invite)
+      invite.update_columns(family_id: current_user.family.id,
+                            invited_by_id: current_user.id)
+    end
+
+    def message(invite)
+      "#{current_user['first_name']} #{current_user['last_name']} added you as family\
+        member on FellowDriver. Click the link below to accept the invitation:\
+        http://localhost:3000/api/users/invitation/accept?invitation_token=#{invite.raw_invitation_token}"
```

Comment:
didn't noticed early, but use heredoc for multi line strings

Reply:
done